FirebaseMLModelDownloader/Sources/ModelDownloader.swift

```diff
@@ -406,11 +389,13 @@ extension ModelDownloader {
         switch downloadModelInfoResult {
         /// New model info was downloaded from server.
         case let .modelInfo(remoteModelInfo):
+          /// Progress handler for model file download.
```

Comment:
Nit: did you mean for this to be 3 1/1 or 2? It does work as 3 in this case since it's over a local variable, but wanted to check.

Reply:
It wasn't intentional either way :) But I fixed it to be `//`, since I meant it as a comment.